package.json

```diff
@@ -83,6 +83,7 @@
     "hash.js": "1.1.3",
     "he": "0.5.0",
     "html-loader": "0.4.0",
+    "html-to-react": "1.3.1",
```

Comment:
Do we really need this one? Was there some problem inserting content? We generate the privacy policy so it should be safe.

Note that if we do need to add a new package, the shrinkwrap must also be updated.

Reply:
> Was there some problem inserting content? We generate the privacy policy so it should be safe.

Do you mean using the dangerouslySetInnerHTML attribute? It's ok to me. The thing is that we have an eslint rule for this, but we could avoid it adding a comment at the top of the file.

Are you agree with this? Because I am :-p

Reply:
added here 76c93dc

Reply:
> Do you mean using the dangerouslySetInnerHTML attribute?

Yes, it's ok for us to do this with trusted content.